batcher/aligned-batcher/src/lib.rs

```diff
@@ -672,7 +672,20 @@ impl Batcher {
             return Ok(());
         };
 
-        if !self.check_min_balance(proofs_in_batch + 1, user_balance) {
+        let msg_max_fee = nonced_verification_data.max_fee;
+        let Some(user_min_fee) = batch_state_lock.get_user_min_fee(&addr).await else {
+            std::mem::drop(batch_state_lock);
+            send_message(
+                ws_conn_sink.clone(),
+                SubmitProofResponseMessage::InvalidNonce,
+            )
+            .await;
+            self.metrics.user_error(&["invalid_nonce", ""]);
```

Comment:
Is the err alright? shouldn't be something related to the balance or max fee?

Reply:
This was from staging. Lets address it in a separate issue/pr.

Reply:
Issue created: https://github.com/yetanotherco/aligned_layer/issues/1412